docs/image-sizes.md

```diff
@@ -0,0 +1,40 @@
+# Images Sizes
+
+You can define additional image thumbnail sizes with `add_image_size`.
+
+The custom thumbnail sizes should be declared in the callback function to the `after_setup_theme` action.
+
+Example:
+```php
+add_action( 'after_setup_theme', __NAMESPACE__ . '\\theme_setup' );
+function theme_setup() {
+    add_image_size( 'category-thumb', 300, 9999 ); // 300 pixels wide (and unlimited height)
+}
+```
+
+You can find more info at the [WordPress Developer Reference](https://developer.wordpress.org/reference/functions/add_image_size/) and [WordPress developer guide](https://developer.wordpress.org/themes/functionality/featured-images-post-thumbnails/#add-custom-featured-image-sizes)
+
+## Hooks and filters
+
+You can add custom image sizes by hooking in on this filter.
+
+### `image_size_names_choose`
+
+Runs before rendering the list of available thumbnail sizes in the sidebar of the media edit screen. Used in conjuction with `add_image_size` 
+
+```php
+    add_filter( 'image_size_names_choose', function ( array $sizes ) {
```

Comment:
No need to indent this. Seems to be using spaces for indentation too, we use tabs in all the other code examples so best to stay consistent.